cnxarchive/sql/migrations/20160923184915_correct_uuid_index.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+
+
+def up(cursor):
+    # Create correct index, drop less useful one
+    cursor.execute("""
+    CREATE INDEX modules_uuid_txt_version_idx ON modules
```

Comment:
fixed (with --fixup and --autosquash, so history is modified --force-with-lease deployed)